services/director/src/simcore_service_director/registry_proxy.py

```diff
@@ -410,11 +410,17 @@ async def get_image_details(
     if not labels:
         return image_details
     for key in labels:
-        if not key.startswith("io.simcore."):
+        if not key.startswith("io.simcore.") and not key.startswith(
+            "simcore.service."
+        ):  # Keeping "simcore.service." adds additonally input_paths, output_paths, state_paths
             continue
         try:
             label_data = json.loads(labels[key])
             for label_key in label_data:
+                if isinstance(
+                    label_key, dict
```

Comment:
- AFAIK a dict can only use has hashable types as keys, i.e.
```
>>> d = {}
>>> d[d] = 33
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unhashable type: 'dict'
```

so the only case i can imagine is that `label_data` is a `list[dict| ... ]` or some equivalent iterable that might return dicts.
Is this condition ONLY applicable to ` "simcore.service." ` ?